csharp-package/brainflow/brainflow/board_shim.cs

```diff
@@ -397,6 +397,29 @@ public static int[] get_temperature_channels (int board_id)
             return result;
         }
 
+        /// <summary>
+        /// get resistance channels for this board
+        /// </summary>
+        /// <param name="board_id"></param>
+        /// <returns>array of row nums</returns>
+        /// <exception cref="BrainFlowException">If this board has no such data exit code is UNSUPPORTED_BOARD_ERROR</exception>
+        public static int[] get_resistance_channels(int board_id)
+        {
+            int[] len = new int[1];
+            int[] channels = new int[512];
```

Comment:
What about here? Allocating on the heap but these arrays are not being returned?

Reply:
Nevermind, this is c#. I'm not 100% awake yet 😄